force-app/main/default/aura/defaultTokens/defaultTokens.tokens

```diff
@@ -0,0 +1,3 @@
+<aura:tokens extends="force:base">
```

Comment:
Is this to simply make sure we have the tokens bundle in the project?

Reply:
yes please, so that we can use standard tokens in Aura.